File: resc/WebApplication/src/modules/application/models/application.dto.ts
import { ActualEducationDto } from "./actual-education.dto";
import { ContractDto } from "./contract.dto";
import { EmployerDto } from "./employer.dto";
import { StudentDto } from "./student.dto";
import { UniversityDto } from "./university.dto";

export class ApplicationDto {
  student: StudentDto;
  university: UniversityDto;
  employer: EmployerDto;
  contract: ContractDto;
  actualEducation: ActualEducationDto;
  isDraft: boolean;

  constructor() {
    this.student = new StudentDto();
    this.university = new UniversityDto();
    this.employer = new EmployerDto();
    this.contract = new ContractDto();
    this.actualEducation = new ActualEducationDto();
  }
}

File: resc/Resc.Application/Applications/Dtos/Create/ApplicationDto.cs
namespace Resc.Application.Applications.Dtos.Create
{
    public class ApplicationDto
    {
        public StudentDto Student { get; set; }
        public UniversityDto University { get; set; }
        public EmployerDto Employer { get; set; }
        public ContractDto Contract { get; set; }
        public ActualEducationDto ActualEducation { get; set; }
        public bool? IsDraft { get; set; }
    }
}
